### ISSUE TITLE
Typescript not up to date?

### ISSUE THREAD

**Issue by ubergesundheit** [cristianbote/goober#156]
I tried goober 2.0.1 and noticed that the signature of the `setup` function is not the same in the types. The types are missing the `theme` argument.

Are there any plans to update the types?

**Comment by cristianbote** [comment]
Hey @ubergesundheit, thanks for opening this issue.

Indeed 😞  that's a bug. Gonna whip out a fix in a second.

**Comment by cristianbote** [comment]
Fixed in `2.0.2`. Let me know if you are running into more issues. Thank you so much for opening this one!

**Comment by ubergesundheit** [comment]
Oh wow, that was fast <3 Thank you very much!

Now when trying to use the theme in a styled component like this:

```js
const KioskWrapper = styled("div")`
  width: 1920px;
  height: 1080px;
  overflow: hidden;
  color: ${(p) => {
    console.log({ p });
    return p.theme.colors.primary;
  }};
`;
```

Typescript complains `Property 'theme' does not exist on type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'`
